webui/react/src/components/Markdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tabs } from 'antd';
+import { default as MarkdownViewer } from 'markdown-to-jsx';
+import React from 'react';
+
+import css from './Markdown.module.scss';
+import Spinner from './Spinner';
+
+const { TabPane } = Tabs;
+const MonacoEditor = React.lazy(() => import('components/MonacoEditor'));
+
+interface Props {
+  editing?: boolean;
+  height?: string;
+  markdown: string;
+  onChange?: (editedMarkdown: string) => void;
+}
+
+const Markdown: React.FC<Props> = ({ editing=false, height='80vh', markdown, onChange }: Props) => {
+  return (
+    editing ?<Tabs>
+      <TabPane key="1" tab="Edit">
```

Comment:
**chore:** this is not immediately needed but planning ahead, we might want to expose the editing state in the URL of the respective pages. To better prep for that, it'll be good to have an `enum TabType` and use that as the key. Take a look at `TrialDetails.tsx` for an example.